documentation/docs/installation/source/windows.md

```diff
@@ -3,9 +3,7 @@
 !!! info "Recommended for Developers Only"
     The [downloadable installer](/installation/download/windows/) is the easiest way to run Hummingbot. Installing from source is only recommended for developers who want to access and modify the software code.
 
-Coming soon.
-
-<!-- The Hummingbot code base is designed and optimized for UNIX-based systems such as macOS and Linux. We recommend that Windows users either:
+The Hummingbot code base is designed and optimized for UNIX-based systems such as macOS and Linux. We recommend that Windows users either:
 
```

Comment:
* Follow [Windows Binary Installation Guide](https://docs.hummingbot.io/installation/download/windows/) to download and install using setup.exe installer binary.